docs/source/tutorial_optimization.md

```diff
@@ -0,0 +1,848 @@
+---
+jupytext:
+  text_representation:
+   format_name: myst
+jupyter:
+  jupytext:
+    cell_metadata_filter: -all
+    formats: ipynb,myst
+    main_language: python
+    text_representation:
+      format_name: myst
+      extension: .md
+      format_version: '1.3'
+      jupytext_version: 1.11.2
+kernelspec:
+  display_name: Python 3
+  name: python3
+---
+
+```{try_on_binder}
+```
+
+```{code-cell}
+:tags: [remove-cell]
+:load: myst_code_init.py
+```
+
+# Tutorial: Model order reduction for PDE-constrained optimization problems
+
+A typical application of model order reduction for PDEs are
+PDE-constrained parameter optimization problems. These problems aim to
+find a local minimizer of an objective functional depending on an
+underlying PDE which has to be solved for all evaluations.
+A prototypical example of a PDE-constrained optimization problem can be defined
+in the following way. For a physical domain {math}`\Omega \subset \mathbb{R}^d` and a
+parameter set {math}`\mathcal{P} \subset \mathbb{R}^P`, we want to find
+a solution of the minimization problem
+
+```{math}
+\min_{\mu \in \mathcal{P}} J(u_{\mu}, \mu),  \tag{P.a}
+```
+
+where {math}`u_{\mu} \in V := H^1_0(\Omega)` is the solution of
+
+```{math}
+\begin{equation} \label{eq:primal}
+a_{\mu}(u_{\mu}, v) = f_{\mu}(v) \qquad \forall \, v \in V \tag{P.b}.
+\end{equation}
+```
+
+The equation {math}`\eqref{eq:primal}` is called the primal
+equation and can be arbitrarily complex. MOR methods in the context of
+PDE-constrained optimization problems thus aim to find a surrogate model
+of {math}`\eqref{eq:primal}` to reduce the computational costs of
+an evaluation of {math}`J(u_{\mu}, \mu)`.
+
+If there exists a unique solution {math}`u_{\mu}` for all
+{math}`\mu \in \mathcal{P}`, we can rewrite (P) by using the so-called
```

Comment:
What about using `eqref` here?

Reply:
There is no label for doing that since it is a sum of two equations, but is probably possible somehow

Reply:
Maybe you can simply write `(P.a) and (P.b)` instead of `(P)`? Of course using `eqref`. I think just replacing `(P)` by `(P.a)` would also make sense here, since it seems to be clear that one still needs `(P.b)`. @TiKeil, what do you think?